src/index.html

```diff
@@ -9,8 +9,106 @@
     <title>GeneXus Web Controls Library test page</title>
     <script type="module" src="/build/gx-web-controls.esm.js"></script>
     <script nomodule src="/build/gx-web-controls.js"></script>
+    <style>
```

Comment:
Let's leave index.html empty. It's great if you want to create a sample page, but it would be better if you create a new html page inside the `samples` directory, under the root directory.

Reply:
yes! I forgot to do that. thx for remind it me! :D

Reply:
Ready! Fixed in f3a1266